feat: MinIO Deployment and S3 to S3 tutorial (#179)
eclipse-tractusx/tutorial-resources: mxd/docs/File Transfer S3 to S3.md

## 12. Get Transfer
Just wait for few seconds and check the transfer state.  
Please replace transfer process id before running command.
```shell
curl --location 'http://localhost/bob/management/v2/transferprocesses/<TRANSFER_PROCESS_ID>' \
--header 'X-Api-Key: password'
```

It should return below response:
```json
{
  "@id": "40fdb511-68f2-453f-bd9b-8e5e728508aa",
  "@type": "TransferProcess",
  "correlationId": "40fdb511-68f2-453f-bd9b-8e5e728508aa",
  "state": "COMPLETED",
  "stateTimestamp": 1701425542142,
  "type": "CONSUMER",
  "assetId": "20",
  "contractId": "48bad8fd-e24d-40e9-91d3-6d7f82693989",
  "callbackAddresses": [],
  "dataDestination": {
    "@type": "DataAddress",
    "secretAccessKey": "bobawssecret",
    "accessKeyId": "bobawsclient",
    "endpointOverride": "http://bob-minio:9000",
    "region": "us-east-1",
    "type": "AmazonS3",
    "keyName": "alice-test-document.txt",
    "bucketName": "bob-bucket"
  },
  "connectorId": "BPNL000000000001",
  "@context": {}
}
```
> Please note, `state` should be `COMPLETED`. If it is `TERMINATED`, it means, transfer has been failed. Please check logs for more details.

## 13. Validate Transfer

> Now, we can validate whether file has been transferred in `bob-bucket` or not.  

Reviewer: ```suggestion
Now we can validate that file has been transferred to `bob-bucket`.  
```

> Once transfer is complete, file `alice-test-document.txt` should be present in `bob-bucket`.

Reviewer: ```suggestion
Once transfer is complete, `alice-test-document.txt` should be present in `bob-bucket`.
```
